frontend/simple/views/Join.vue

```diff
@@ -0,0 +1,163 @@
+<template>
+    <section class="section full-screen">
+      <div class="column is-10" >
+        <div class="columns" v-if="contract">
+          <div class="column is-one-half">
+            <div class="center ">
+              <div class="subtitle is-3"><i18n>You have been Invited to join</i18n></div>
+              <div class="title is-1">{{contract.groupName}}</div>
+            </div>
+            <div class="panel">
+              <div class="panel-block">
+                <div style="display: inline;">
+                  <strong>Shared Values: </strong> {{contract.sharedValues}}
+                </div>
+              </div>
+              <table class="panel-block notification is-warning">
+                <strong>${{contract.incomeProvided}} Monthly Income</strong>
```

Comment:
Missing <i18n> here and above. Might be more elsewhere, double-check, triple-check, quadruple-check.

There's probably a regex you can use to automatically find them. e.g. searching for `\w+` *not* preceeded by `<i18n>`